views/index.ejs

```diff
@@ -43,37 +56,51 @@
           <section id="main-content">
             <header>
               <% if (!isWebRTCVersion) { %>
-                <h1>OpenTok Demo by</h1>
+              <h1>OpenTok Demo by</h1>
               <% } else { %>
-                <h1>Vonage Free Conferencing</h1>
+              <h1>Vonage Free Conferencing</h1>
               <% } %>
-              <img src="images/vonage-logo.svg" alt="vonage transparent logo">
+              <img src="images/vonage-logo.svg" alt="vonage transparent logo" />
               <% if (isWebRTCVersion) { %>
-                <p>
-                  Free to use for everyone who unexpectedly needs to work from home, etc.
-                </p>
+              <p>
+                Free to use for everyone who unexpectedly needs to work from
+                home, etc.
+              </p>
               <% } %>
             </header>
             <form>
               <div>
-                <input id="room" class="text-input required" data-wd="roomname" maxlength="25">
+                <input
+                  id="room"
+                  class="text-input required"
+                  data-wd="roomname"
+                  maxlength="25"
+                />
                 <label id="room-label" class="visited">Meeting name</label>
                 <p class="error-room error-text">
                   <i data-icon="warning"></i>
                   Please enter a meeting name.
                 </p>
               </div>
               <div>
-                <input id="user" class="text-input" data-wd="username" maxlength="25">
+                <input
+                  id="user"
+                  class="text-input"
+                  data-wd="username"
+                  maxlength="25"
+                />
                 <label id="user-label">Your name</label>
               </div>
             </form>
+            <button id="enter" data-wd="enterroom">Join Meeting</button>
```

Comment:
This is basically a duplicate of the already existing button that we currently have implemented. This doesn't seem to be intentional. Can you please fix this? Thanks!